config/app.php

```diff
@@ -215,7 +215,7 @@
         'Validator' => Illuminate\Support\Facades\Validator::class,
         'View' => Illuminate\Support\Facades\View::class,
         'Image' => Intervention\Image\Facades\Image::class,
-
+        'photo' => \Rogue\Models\Photo::class,
```

Comment:
@DFurnes ok I will merge! I wasn't sure how to do the mapping you mentioned [here](https://github.com/DoSomething/rogue/pull/150#discussion_r106202388) and thought this might be the way to do it? Should I make an issue for it?